modules/tensor_mechanics/src/userobjects/TensorMechanicsPlasticModel.C

```diff
@@ -189,15 +201,15 @@ TensorMechanicsPlasticModel::returnMap(const RankTwoTensor & trial_stress, const
   // example of checking Kuhn-Tucker
   std::vector<Real> dpm(numberSurfaces(), 0);
   for (unsigned sf = 0 ; sf < numberSurfaces() ; ++sf)
-    if (!KuhnTuckerSingleSurface(yf[sf], dpm[sf]))
+    if (!KuhnTuckerSingleSurface(yf[sf], dpm[sf], 0))
```

Comment:
I've decided not to make a default value for dpm_tol in this function.  It is good to have a tolerance value entered as a non-default argument i think.